man/systemd.network.xml

```diff
@@ -193,6 +193,14 @@
           This happens when multicast routing is enabled.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>IPv6LinkLocalAddressGenerationMode=</varname></term>
+        <listitem>
+          <para>Specifies how IPv6 link local address is generated. Takes one of <literal>eui64</literal>,
+          <literal>none</literal>, <literal>stable-privacy</literal> and <literal>random</literal>.
+           When unset, the kernel's default will be used.</para>
```

Comment:
How does this interact with LinkLocalAddressing= and does "none" mean? E.g. when LinkLocalAdressing=ipv6 and IPv6LinkLocalAddressGenerationMode=none.

Reply:
The backward compatibility `LinkLocalAdressing=ipv6 ` is honoured and takes precedence May be update in the man too.